spdx/relationship.py

```diff
@@ -112,3 +118,5 @@ def validate_relationship(self, messages):
                 "Relationship type must be one of the constants defined in "
                 "class spdx.relationship.Relationship"
             )
```

Comment:
Why do we use these two functions if the `validate` method only calls the `validate_relationship` method? IMHO we should only use the `validate_relationship` method.